apps/web/src/components/applicant/ApplicantProfile.tsx

```diff
@@ -84,6 +75,31 @@ export const ApplicantProfile = () => {
                 .join(', ')
             }
           />
+          <DetailsItem title='Assigned To'>
+            <div>
+              <span className='border border-gray-200 bg-bcGrayLabel text-white rounded text-xs px-2 py-0.5 mr-1'>
+                HMBC
```

Comment:
curiosity questions, is showing HMBC necessary once they're assigned to a HA? and should HMBC/ MoH be able to see who they're assigned too, maybe not show the name, but could be an easier way to view what HA's the applicant is apart of, especially since there's no search for licensing/ registration?

Reply:
no no, I meant from a HMBC perspective, for example if I log in as a HA assign a user to me, you can see `HMBC` and the HA the user is from, then when you sign back in as HMBC you only see HMBC.  Just a thought for more clarity for HMBC, not a blocker though.

Reply:
It's the same for HMBC and HAs. Recruiter is meant to be used only for HAs, so I don't think we need to let HMBC know each HA's recruiters.